README.md

```diff
@@ -268,6 +300,28 @@ kubectl apply -f $GOPATH/src/github.com/kubeedge/kubeedge/build/node.json
 
 Run Edge
 
+#### Method A
+
+This method will guide you to deploy the edge part running in docker
+container, so make sure that docker engine listening on
+`/var/run/docker.sock` which will then mount into the edge container.
+
+Before start the edge part container, check the content of this script
```

Comment:
nit: start -> starting

Reply:
Thanks. I will correct the points mentioned above.